cpp/pipelines/add_subsystem_tags.cc

```diff
@@ -425,27 +424,100 @@ void AddSubsystemTags(MARC::Reader * const marc_reader, MARC::Writer * const mar
 }
 
 
+void ExtractAuthors(MARC::Reader * const marc_reader, std::map<std::string, std::set<std::string>> * authors,
```

Comment:
`* authors` should be `* const authors`

Reply:
The authors map can get very large, so it might be better to use std::unordered_map for performance reasons if we do not need any guaranteed order.